### PR TITLE
Feature: support login to node groups

### PR DESCRIPTION
**Pull request by anelhms**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,9925 +1,8 @@
 {
   "name": "node-red-contrib-kolibri",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```
Comment (anelhms) [inline package-lock.json:6]: Not sure why this file changed this much

File: tsconfig.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "noEmitOnError": true,
     "sourceMap": true,
     "outDir": "dist",
-    "rootDir": "./",
+    "rootDir": "src",
```
Comment (anelhms) [inline tsconfig.json:14]: should perhaps have been a separete commit (?)

File: package-lock.json (hunk 1 above)
Comment (arbue) [reply package-lock.json:6]: you need to update your npm to version >7

File: tsconfig.json (hunk 2 above)
Comment (arbue) [reply tsconfig.json:14]: This is fine.

File: src/nodes/kolibri-broker/kolibri-broker.html/index.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ RED.nodes.registerType<KolibriBrokerEditorNodeProperties, KolibriBrokerEditorNod
         name: { value: '', required: true },
         broker: { value: '', required: true, validate: RED.validators.regex(/(?:^|[ \t])((?:localhost|[\w-]+(?:\.[\w-]+)+)(\/\S*)?)/i) },
         port: { value: 443, required: true, validate: RED.validators.number() },
+        path: { value: '/', required: false },
```
Comment (arbue) [inline src/nodes/kolibri-broker/kolibri-broker.html/index.ts:29]: this should be a required value

File: src/nodes/kolibri-broker/kolibri-broker.ts [hunk 4]
```diff
@@ -49,6 +50,7 @@ const nodeInit: NodeInitializer = (RED): void => {
         this.broker = config.broker;
         this.port = config.port;
         this.project = config.project;
+        this.path = config.path;
```
Comment (arbue) [inline src/nodes/kolibri-broker/kolibri-broker.ts:53]: node-red seams not to pass the default values in here. So we need to ensure this has a default value e.g. '/' if the path is undefined.

**Review by arbue — APPROVED** [review]

**Pull request merged**